runtime/interface.go

```diff
@@ -60,6 +60,10 @@ type Interface interface {
 	GetComputationLimit() uint64
 	// DecodeArgument decodes a transaction argument against the given type.
 	DecodeArgument(b []byte, t cadence.Type) (cadence.Value, error)
+	// GetCurrentBlockHeight returns the current block height.
+	GetCurrentBlockHeight() uint64
+	// GetBlockAtHeight returns the block at the given height.
+	GetBlockAtHeight(height uint64) (hash [32]byte, timestamp int64, exists bool)
```

Comment:
Maybe we can define a `cadence.Block` type for this instead of returning the parts as individual return values